content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -284,6 +291,20 @@ public void init() {
 				.filter(e -> e.name().equalsIgnoreCase(serverConfigurationService.getString("turnitin.oc.auto_exclude_self_matching_scope")))
 				.findAny().orElse(AUTO_EXCLUDE_SELF_MATCHING_SCOPE.GROUP).name();
 		log.info("Exclude Scope: " + autoExcludeSelfMatchingScope);
+		
+		// Find any permission overrides, if not set, set value to null to skip overrides
+		mayViewSubmissionFullSourceOverrideStudent = StringUtils.isNotEmpty(serverConfigurationService.getString("turnitin.oc2.may_view_submission_full_source.student")) 
```

Comment:
is the `StringUtils.isNotEmpty` check actually needed? seems like calling getBoolean with a default would be sufficient?

Reply:
I wanted three options: null, false, true, where null means to not set the override (because you could override with false or true)

Reply:
the `getBoolean` call does not allow for a default parameter which I would have used `null`

Reply:
ok understood